developer/src/kmc-copy/test/fixtures/projects/alephwithbeth/build/alephwithbeth.keyboard_info

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "alephwithbeth",
+  "name": "Aleph With Beth",
+  "license": "mit",
+  "isRTL": true,
+  "authorName": "Cameron Briggs",
+  "description": "<p>alephwithbeth keyboard</p>",
+  "languages": {
+    "he": {
+      "examples": [],
+      "languageName": "Hebrew",
+      "displayName": "Hebrew"
+    }
+  },
+  "lastModifiedDate": "2024-06-11T18:56:16.000Z",
+  "packageFilename": "alephwithbeth.kmp",
+  "packageFileSize": 244587,
+  "jsFilename": "alephwithbeth.js",
+  "jsFileSize": 23533,
+  "packageIncludes": [
+    "documentation",
+    "welcome",
+    "visualKeyboard"
+  ],
+  "version": "1.0.6",
+  "encodings": [
+    "unicode"
+  ],
+  "platformSupport": {
+    "windows": "full",
+    "macos": "full",
+    "linux": "full",
+    "desktopWeb": "full",
+    "ios": "full",
+    "android": "full",
+    "mobileWeb": "full"
+  },
+  "minKeymanVersion": "9.0",
+  "sourcePath": "experimental/a/alephwithbeth",
+  "helpLink": "https://help.keyman.com/keyboard/alephwithbeth"
+}
```

Comment:
Confirming you wanted to check in all the test/fixtures/ build/ artifacts?

Reply:
Yes, these are needed for these tests.